vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1126,117 +1162,109 @@ public void failure(VimeoError error) {
      * Activate a picture resource
      *
      * @param uri The Uri that is found in the PictureResource returned from
-     *            {@link #createPictureResource(String, ModelCallback)}
+     *            {@link #createPictureResource(String, VimeoCallback)}
      */
     @Nullable
-    public Call<Object> activatePictureResource(String uri, ModelCallback callback) {
+    public Call<PictureCollection> activatePictureResource(String uri, VimeoCallback<PictureCollection> callback) {
         if (uri == null || uri.trim().isEmpty()) {
             callback.failure(new VimeoError("uri cannot be empty!"));
             return null;
         }
-        HashMap<String, Object> parameters = new HashMap<>();
+        final HashMap<String, Object> parameters = new HashMap<>();
         parameters.put(Vimeo.PARAMETER_ACTIVE, true);
-        Call<Object> call = mVimeoService.edit(getAuthHeader(), uri, parameters);
-        call.enqueue(getRetrofitCallback(callback));
+        final Call<PictureCollection> call = mVimeoService.editPictureCollection(getAuthHeader(), uri, parameters);
+        call.enqueue(callback);
         return call;
     }
-
-    /**
-     * Delete a picture resource
-     * This is helpful if your activation or upload step fails - you can delete the resource allocated
-     *
-     * @param uri The Uri that is found in the PictureResource returned from
-     *            {@link #createPictureResource(String, ModelCallback)}
-     */
-    public void deletePictureResource(String uri, ModelCallback callback) {
```

Comment:
I would say since this method provides nothing over the fall through method other than throwing a failure with an empty URI it should be deleted. I'm on board with deleting it. It would be nice if we had specific delete methods in the future though only for content which is deletable and instead of taking in a URI they would accept the object to delete and then grab the URI from the object.